dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/AsyncRpcResult.java

```diff
@@ -121,10 +149,14 @@ public void setResultFuture(CompletableFuture<Result> resultFuture) {
     public Result getRpcResult() {
```

Comment:
I think the whole method can be simplified by 

```
public Result getRpcResult() {
        try {
            if (resultFuture.isDone()) {
                return resultFuture.get();
            }
        } catch (Exception e) {
            // This should never happen;
            logger.error("Got exception when trying to fetch the underlying result of AsyncRpcResult.", e);
        }
        return new RpcResult();
    }
```

what do you say?

Reply:
+1

Reply:
Well done. I think now I should spend some time to learn how to write clean neat code  😂.